addons/mail/static/src/core/public_web/discuss_content.xml

```diff
@@ -27,7 +27,7 @@
                     </div>
                     <AutoresizeInput
                         className="'o-mail-DiscussContent-threadName fw-bold flex-shrink-1 py-0 fs-4'"
-                        enabled="thread.is_editable or thread.channel_type === 'chat'"
+                        enabled="thread.is_editable or thread.channel?.channel_type === 'chat'"
```

Comment:
Ok for another PR but channel related code should be moved as discuss folder override.